### PR TITLE
VZ-7432 psrcrtl first pass

### PR DESCRIPTION
**Pull request by stevshap**
First pass for the psrctl CLI framework
Make file updated and will install the CLI with `make cli`
CLI can be run using `psrctl [command]`
Only `version` command is currently implemented

### CODE REVIEW

File: tools/psr/Makefile [hunk 1]
```diff
@@ -70,6 +84,20 @@ run-example-oam: kind-load-image
 run-example-k8s: kind-load-image
 	helm upgrade --install psr manifests/charts/worker --set appType=k8s --set imageName=${DOCKER_IMAGE_FULLNAME}:${DOCKER_IMAGE_TAG}
 
+#
+# run the CLI
+#
+.PHONY: run
```
Comment (pfmackin) [inline tools/psr/Makefile:90]: change to run-cli

File: tools/psr/Makefile [hunk 2]
```diff
@@ -70,6 +84,20 @@ run-example-oam: kind-load-image
...
+run:
+	$(GO) run ${GOPATH}/src/${CLI_DIR}/main.go
+
+#
+# build the CLI
+#
+.PHONY: cli
```
Comment (pfmackin) [inline tools/psr/Makefile:97]: change to build-cli

File: tools/psr/Makefile [hunk 3]
```diff
@@ -70,6 +84,20 @@ run-example-oam: kind-load-image
...
 
+#
+# run the CLI
+#
+.PHONY: run
+run:
+	$(GO) run ${GOPATH}/src/${CLI_DIR}/main.go
+
+#
+# build the CLI
```
Comment (pfmackin) [inline tools/psr/Makefile:95]: install the CLI

File: tools/psr/Makefile [hunk 4]
```diff
@@ -28,7 +30,9 @@ BUILD_VERSION:=$(shell grep verrazzano-development-version ${MAKEFILE_DIR}/../..
 BUILD_DATE:=$(shell date +"%Y-%m-%dT%H:%M:%SZ")
 
 DIST_DIR:=dist
+ENV_NAME=psr
```
Comment (mbenmclean) [inline tools/psr/Makefile:33]: Why do we need this?

File: tools/psr/psrctl/cmd/run/run.go [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+	cmdhelpers "github.com/verrazzano/verrazzano/tools/vz/cmd/helpers"
+	"github.com/verrazzano/verrazzano/tools/vz/pkg/helpers"
+)
+
+const (
+	CommandName = "run"
+	helpShort   = "Run a PSR test scenario"
+	helpLong    = `The command 'run' executes a PSR test scenario consisting of one or more use cases`
+	helpExample = `
+psrctl run psr-1`
```
Comment (mbenmclean) [inline tools/psr/psrctl/cmd/run/run.go:17]: psrctl run scenario-1

**Review by mbenmclean — DISMISSED** [review]

File: tools/psr/Makefile (hunk 4 above)
Comment (stevshap) [reply tools/psr/Makefile:33]: tbh not sure, pulled it from the vz cli

**Review by mbenmclean — APPROVED** [review]

**Pull request merged**